### ISSUE TITLE
entities URIs must not escape the single quote (')

### ISSUE THREAD

**Issue by ziodave** [insideout10/wordlift-plugin#51]
We received a SPARQL query with an URI with an escape (\) before a single quote:

    <http://data.redlink.io/91/be5/post/Some_people_talkin\'>

The escape is causing an error, it should be changed to

    <http://data.redlink.io/91/be5/post/Some_people_talkin'>

Error SPARQL:

```SPARQL
PREFIX geo: <http://www.w3.org/2003/01/geo/wgs84_pos#>
PREFIX dct: <http://purl.org/dc/terms/>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX owl: <http://www.w3.org/2002/07/owl#>
PREFIX schema: <http://schema.org/>
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
DELETE { <http://data.redlink.io/91/be5/entity/Onofrio> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/entity/Onofrio> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/entity/Palermo> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/entity/Palermo> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/entity/Mount_Etna> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/entity/Mount_Etna> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/post/My_trip_on_the_Etna> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/post/My_trip_on_the_Etna> schema:url
?u . };
DELETE { <http://data.redlink.io/91/be5/entity/North_Pole> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/entity/North_Pole> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/post/Where_do_I_want_to_go_> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/post/Where_do_I_want_to_go_> schem
a:url ?u . };
DELETE { <http://data.redlink.io/91/be5/entity/Pizza> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/entity/Pizza> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/post/Some_people_talkin\'> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/post/Some_people_talkin\'> schema:ur
l ?u . };
DELETE { <http://data.redlink.io/91/be5/page/Blog_map> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/page/Blog_map> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/post/A_post_about_friends> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/post/A_post_about_friends> schema:ur
l ?u . };
DELETE { </page/Sample_Page> schema:url ?u . } WHERE  { </page/Sample_Page> schema:url ?u . };
INSERT DATA {  <http://data.redlink.io/91/be5/entity/Onofrio> schema:url <http://localhost/wp4wl3/?entity=onofrio> .
 <http://data.redlink.io/91/be5/entity/Palermo> schema:url <http://localhost/wp4wl3/?entity=palermo> .
 <http://data.redlink.io/91/be5/entity/Mount_Etna> schema:url <http://localhost/wp4wl3/?entity=mount-etna> .
 <http://data.redlink.io/91/be5/post/My_trip_on_the_Etna> schema:url <http://localhost/wp4wl3/?p=46> .
 <http://data.redlink.io/91/be5/entity/North_Pole> schema:url <http://localhost/wp4wl3/?entity=north-pole> .
 <http://data.redlink.io/91/be5/post/Where_do_I_want_to_go_> schema:url <http://localhost/wp4wl3/?p=38> .
 <http://data.redlink.io/91/be5/entity/Pizza> schema:url <http://localhost/wp4wl3/?entity=pizza> .
 <http://data.redlink.io/91/be5/post/Some_people_talkin\'> schema:url <http://localhost/wp4wl3/?p=21> .
 <http://data.redlink.io/91/be5/page/Blog_map> schema:url <http://localhost/wp4wl3/?page_id=18> .
 <http://data.redlink.io/91/be5/post/A_post_about_friends> schema:url <http://localhost/wp4wl3/?p=5> .
 </page/Sample_Page> schema:url <http://localhost/wp4wl3/?page_id=2> .
 };
```

**Comment by ziodave** [comment]
Still failing

```sparql
PREFIX geo: <http://www.w3.org/2003/01/geo/wgs84_pos#>
PREFIX dct: <http://purl.org/dc/terms/>
PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
PREFIX owl: <http://www.w3.org/2002/07/owl#>
PREFIX schema: <http://schema.org/>
PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
DELETE { <http://data.redlink.io/91/be5/post/L\'Aquila_è_la_città_bella> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/post/L\'Aquila_è_la_città_bella> schema:url ?u . };
DELETE { <http://data.redlink.io/91/be5/page/Blog_map> schema:url ?u . } WHERE  { <http://data.redlink.io/91/be5/page/Blog_map> schema:url ?u . };
DELETE { </page/Sample_Page> schema:url ?u . } WHERE  { </page/Sample_Page> schema:url ?u . };
INSERT DATA {  <http://data.redlink.io/91/be5/post/L\'Aquila_è_la_città_bella> schema:url <http://localhost/wp4wl3/?p=76> .
 <http://data.redlink.io/91/be5/page/Blog_map> schema:url <http://localhost/wp4wl3/?page_id=18> .
 </page/Sample_Page> schema:url <http://localhost/wp4wl3/?page_id=2> .
 };
```

**Comment by ziodave** [comment]
Merged PR #52
